ext/web/lib.deno_web.d.ts

```diff
@@ -68,7 +68,10 @@ declare var DOMException: {
   readonly DATA_CLONE_ERR: 25;
 };
 
-/** @category Events */
+/** Specifies the options that can be passed to the constructor of an `Event`
+ * object. The EventInit interface defines three optional properties: `bubbles`, `cancelable`, and `composed`.
```

Comment:
```suggestion
 * object.
```

Seems unnecessary- one can already see this in the interface